editor/plugins/animation_state_machine_editor.cpp

```diff
@@ -1022,6 +1030,10 @@ void AnimationNodeStateMachineEditor::_state_machine_draw() {
 		Vector2 ofs_to = (dragging_selected && selected_nodes.has(tl.to_node)) ? drag_ofs : Vector2();
 		tl.to = (state_machine->get_node_position(tl.to_node) * EDSCALE) + ofs_to - state_machine->get_graph_offset() * EDSCALE;
 
+		if (!(state_machine->get_node_visibility(tl.from_node) & state_machine->get_node_visibility(tl.to_node))) {
```

Comment:
```suggestion
		if (!state_machine->get_node_visibility(tl.from_node) || !state_machine->get_node_visibility(tl.to_node)) {
```
You were using bitwise and here `&` instead of logical and `&&`, and better to simplify the expression